caddyhttp/httpserver/server.go

```diff
@@ -507,16 +507,46 @@ func (s *Server) Stop() error {
 // OnStartupComplete lists the sites served by this server
 // and any relevant information, assuming caddy.Quiet == false.
 func (s *Server) OnStartupComplete() {
+
 	if caddy.Quiet {
 		return
 	}
+
+	s.OutputSiteInfo()
+
+}
+
+func (s *Server) OutputSiteInfo() {
+
+	sitesByPort := make(map[string][]SiteConfig)
 	for _, site := range s.sites {
 		output := site.Addr.String()
-		if caddy.IsLoopback(s.Address()) && !caddy.IsLoopback(site.Addr.Host) {
-			output += " (only accessible on this machine)"
+
+		sitesArray := sitesByPort["port:"+site.Port()]
```

Comment:
Reads better with a space IMO (as english)
```suggestion
		sitesArray := sitesByPort["port "+site.Port()]
```

Reply:
Thanks, I'll change that.  I accidentally submitted 2 PRs so I'm going to close this one.